### PR TITLE
Add sample project for android

### PR DESCRIPTION
**Pull request by gouniLee**
* Add sample for android project
  * includes xliff (v1.2), strings and layout xml files. project.json file.
 * It would work the upcoming latest release android plugin version (1.0.1)
   * https://github.com/iLib-js/ilib-loctool-android-layout/pull/2
   * https://github.com/iLib-js/ilib-loctool-android-resource/pull/4

### CODE REVIEW

**Comment by gouniLee** [comment]
loctool run without errors, but it doesn't write string from layout `t1.xml` in `sample-extracted.xliff` file now

**Comment by ehoogerbeets** [comment]
That's because you don't have any strings to extract from t1.xml!

```
                                                                          android:text="@string/text_Unlimited_Gigabytes_of_Data"
                                                                                                     ^^^^             ^^^ 
                    this means it comes from the string table in strings.xml               this is the unique id
```

To have a string that you can extract, it should be something like:

```
                                                                          android:text="Unlimited Gigabytes of Data"
```

**Comment by ehoogerbeets** [comment]
Pushed an updated t1.xml to show the extracted string. The plugin also fixes up the source file to "resourcify" the string. That is, it moves it automatically to the the strings.xml.

**Comment by gouniLee** [comment]
> Pushed an updated t1.xml to show the extracted string. The plugin also fixes up the source file to "resourcify" the string. That is, it moves it automatically to the strings.xml.

oh.. I see. I checked it updated the `t1.xml` file automatically and generate the `strings-auto.xml file`. and confirmed it generates the resource data properly in `values-ko/strings.xml`

File: android/res/values/strings.xml [hunk 1]
```diff
@@ -13,5 +13,22 @@
...
+    </item>
+  </plurals>
+  <string-array name="days">
+    <item>Sunday</item>
+    <item>Monday</item>
+    <item>Tuesday</item>
+    <item>Wednesday</item>
+    <item>Thursday</item>
+    <item>Friday</item>
+    <item>Satday</item>
```
Comment (ehoogerbeets) [inline android/res/values/strings.xml:31]: Saturday

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**